hackathon_site/hackathon_site/settings/__init__.py

```diff
@@ -122,7 +122,8 @@
 
 REST_FRAMEWORK = {
     "DEFAULT_AUTHENTICATION_CLASSES": [
-        "rest_framework.authentication.SessionAuthentication"
+        "rest_framework.authentication.TokenAuthentication",
+        "rest_framework.authentication.SessionAuthentication",
```

Comment:
I thought we were going with only token auth?

Reply:
The react hardware site still uses session authentication. Having token authentication enabled just gives us another option, and makes `HTTP 401 Unauthorized` possible.